web-frontend/js/load.js

```diff
@@ -44,7 +48,6 @@ langRequest()
   })
   .catch(function (err) {
     langs = ["Error!"];
-    console.log("Error when fetching languages: " + err);
```

Comment:
Any reason to remove this log?

Reply:
my bad! I was doing some simple testing on the frontend and it must've slipped through the cracks. I'll push out another commit that undoes it